tests/test_strategy.py

```
@@ -159,7 +159,7 @@ def action_probability_pairs(draw, min_actions: int = 2, max_actions: int = 10,
         if allow_callables and draw(st.booleans()):
             # Create a callable probability
             probabilities[action_id] = lambda x, p=probability_value: p
-            models[action_id] = DummyZoomingModel.cold_start(dimension=DEFAULT_DIMENSION, cost=lambda x: cost_value)
+            models[action_id] = DummyZooming.cold_start(dimension=DEFAULT_DIMENSION, cost=lambda x: cost_value)
```

Ruff (0.15.7)

[warning] Function definition does not bind loop variable `cost_value`

(B023)